trident/ion_balance.py

```diff
@@ -437,10 +437,10 @@ def add_ion_fraction_field(atom, ion, ds, ftype="gas",
 
     atom = atom.capitalize()
 
-    # if neutral ion field, alias X_p0_ion_fraction to X_ion_fraction field
+    # if neutral ion field, alias X_number_density to X_p0_number_density field
     if ion == 1:
```

Comment:
Since the regular field name for the neutral species now follows the same logic as the other ions, this code block can be made a bit simpler and more readable. Could you try something like this, here and in the other instances in this PR?

```suggestion
    field = "%s_p%d_ion_fraction" % (atom, ion-1)
    if ion == 1:
        alias_field = "%s_ion_fraction" % atom
```

Reply:
Ya, that makes sense. I'll make the change.